src/main/scala/lenthall/spray/SprayCanHttpService.scala

```diff
@@ -75,8 +75,8 @@ object SprayCanHttpService {
     : Future[Unbound] = {
       unbind(duration) andThen {
         case _ =>
-          actorSystem.shutdown()
-          actorSystem.awaitTermination()
+          actorSystem.terminate()
+          Await.ready(actorSystem.whenTerminated, Duration.Inf)
```

Comment:
ToL: boo!!

(could you add a comment justifying why this infinite wait is OK here? And above)